core/unit_test/TestAtomics.hpp

```diff
@@ -510,8 +510,12 @@ TEST(TEST_CATEGORY, atomics) {
   ASSERT_TRUE(
       (TestAtomic::Loop<Kokkos::complex<float>, TEST_EXECSPACE>(100, 3)));
 
-// FIXME_SYCL atomics for large types to be implemented
-#ifndef KOKKOS_ENABLE_SYCL
+// FIXME_SYCL Replace macro by SYCL_EXT_ONEAPI_DEVICE_GLOBAL or remove
+// condition alltogether when possible.
+#if defined(KOKKOS_ENABLE_SYCL) && \
+    !defined(KOKKOS_IMPL_SYCL_DEVICE_GLOBAL_SUPPORTED)
+  if (std::is_same_v<TEST_EXECSPACE, Kokkos::Experimental::SYCL>) return;
```

Comment:
Sorry just saw that.  Why return rather than GTEST_SKIP ?

Reply:
The behavior with `GTEST_SKIP` at the end of a test case is: The tests show as skipped if otherwise successful. If there was a failing check before it shows as failing.
It might be a little confusing to see a test showing as skipped if most of it actually ran. Thus, I don't have a clear preference about using `GTEST_SKIP` here or not and would leave it as-is.

Reply:
Fine by me in the current version.